sass/_ResultsFiltersPreferences.scss

```diff
@@ -150,16 +150,13 @@
     display: inline-block;
     @include clearButton('.coveo-clear');
     @include clickable();
-    & + & {
+    & + .coveo-value {
```

Comment:
Hmm, what does the `& +` part of the selector mean? Usually there should be a selector on either side of the `+`?

Reply:
same as ` .coveo-value  + .coveo-value `
the previous ` & + & ` wasn't working but `& + .coveo-value` works as expected